sky/clouds/cloud.py

```diff
@@ -485,5 +486,25 @@ def query_status(cls, name: str, tag_filters: Dict[str, str],
         """
         raise NotImplementedError
 
+    @classmethod
+    def create_image_from_cluster(cls, name: str, tag_filters: Dict[str, str],
+                                  region: Optional[str],
+                                  zone: Optional[str]) -> str:
+        raise NotImplementedError
+
+    @classmethod
+    def maybe_move_image(cls, image_name: str, source_region: str,
+                         target_region: str, source_zone: Optional[str],
+                         target_zone: Optional[str]) -> str:
+        """Move an image if the target region cannot find it.
```

Comment:
Not sure what this means/if it captures the true intent?

My understanding is GCP's images are global so the "maybe move" part makes sense, but the "if the target region cannot find it" part doesn't?